src/components/settings/SettingsLMSTab/tests/AuthorizationsConfigs.test.tsx

```diff
@@ -242,38 +242,40 @@ describe('Test authorization flows for Blackboard and Canvas', () => {
     expect(mockCanvasFetch).toHaveBeenCalledWith(1);
   });
   test('Canvas config is activated after last step', async () => {
-    renderWithRouter(<SettingsCanvasWrapper />);
-    mockCanvasUpdate.mockResolvedValue({ data: mockCanvasResponseDataActive });
+    setTimeout(async () => {
```

Comment:
[curious] Why is a `setTimeout` necessary here? It feels like a code smell that `setTimeout` would be needed in a test like this when its used React Testing Library.

Reply:
The initial issue was I ran into a "Null: create event" error when running this test file. The problem seemed to be with the last section. Async/await operations weren't working properly there, so I had to use setTimeout to add a small delay in order to clear the current stack.

I have made some changes for this particular test case ([commit](https://github.com/openedx/frontend-app-admin-portal/pull/1009#:~:text=authorization%20test%20file-,db7fdce,-Mashal%2Dm%20force))